### ISSUE TITLE
make the kill tile auto-rescue you if practice is on instead of killing

### ISSUE THREAD

**Issue by def-** [ddnet/ddnet#3024]
make the kill tile auto-rescue you if practice is on instead of killing

as suggested by Дядя Женя on Discord

**Comment by heinrich5991** [comment]
#2221

**Comment by heinrich5991** [comment]
Oops.

**Comment by MEmileTW** [comment]
Why not send the tee back to a spawn tile instead of /rescue location?

**Comment by MysteryZA** [comment]
### I have a few suggestions for this:
- Have this feature as a server command when you are in practice /killtileprotection [int mode] (for example)
### Modes:
1. Use /r when touching a kill tile
2. Kill tile invulnerability

You could also give killtile invulnerability for a short period after using /r to avoid the kill -> /r loop issue.
With this solution people can use the respective modes on maps where it actually makes sense to do so.

**Comment by Jupeyy** [comment]
I don't have any overview about how practise generally is implemented but if kill tiles would simply act like teleporters (/r)
they shouldn't generate any server messages (or kill network packets)

And that would be the easiest to use.

having some meta state where you have to actively use /r sounds rather hard to me. I could ofc be wrong

kill tile invulnerability sounds useful too, maybe generally allow smth similar to `super` in practise.

tl;dr: The feature would probably be useful, I can't give any judgement about code design choices. From usuability i'd prefer this stuff happens automatically :)

**Comment by heinrich5991** [comment]
Duplicate of #2232.
